### ISSUE TITLE
Incorrect resource amounts 

### ISSUE THREAD

**Issue by naeem-bvg** [robertnunn/DRG-Save-Editor#4]
<img width="1108" alt="Screenshot_12" src="https://user-images.githubusercontent.com/17141136/103693619-811dd400-4f91-11eb-9670-2df7d394c9cb.png">

Took to the hex editor not wanting to mess with the app while the overclock bug lurks. I discovered the resources were not in the same order or location, which at least partly explains the odd numbers in the screen shot. This may be because the structure of the save file changes when you overclock or promote or something. I have confirmed these in game.

```
def get_resources(save_bytes):
    # print('getting resources')
    marker = b"OwnedResources"
    start_offset = 101
    offset = 20
    resources = [
        "UNKNOWN",
        "Barley Bulb",
        "Jadiz",
        "Malt Star",
        "Bismor",
        "Enor Pearl",
        "Yeast Cone",
        "Starch Nut",
        "Croppa",
        "Umanite",
        "Magnite",
    ]
```
I don't think Error Cube or Blank Core Matrix are in there at all.

Does the app correctly populate for you?

**Comment by naeem-bvg** [comment]
Just saw your latest push. I believe that will resolve this issue as well.

**Comment by naeem-bvg** [comment]
Fixed in 1.3.3
